miniapps/meshing/mesh-optimizer.cpp

```diff
@@ -173,13 +181,14 @@ int main(int argc, char *argv[])
                   "--no-normalization",
                   "Make all terms in the optimization functional unitless.");
    args.AddOption(&fdscheme, "-fd", "--fd_approximation",
+                  "-no-fd", "--no-fd-approx",
                   "Enable finite difference based derivative computations.");
    args.AddOption(&visualization, "-vis", "--visualization", "-no-vis",
                   "--no-visualization",
                   "Enable or disable GLVis visualization.");
    args.AddOption(&verbosity_level, "-vl", "--verbosity-level",
                   "Set the verbosity level - 0, 1, or 2.");
-   args.AddOption(&adapt_eval, "-ae", "--adaptivity evaluatior",
+   args.AddOption(&adapt_eval, "-ae", "--adaptivity evaluator",
```

Comment:
I just noticed this -- the space in the long-format version of this option is probably not a good idea. In order for it to work right now, it'll have to be quoted: `... "--adaptivity evaluator" 1 ...`.

However, at this point, it's not a big deal because you are using the short version in the sample runs, e.g. `-ae 1`.

Reply:
Fixed it.